modules/kafka/build.gradle

```diff
@@ -7,3 +7,12 @@ dependencies {
     testImplementation 'org.assertj:assertj-core:3.24.2'
     testImplementation 'com.google.guava:guava:23.0'
 }
+
+tasks.japicmp {
+    methodExcludes = [
+        "org.testcontainers.containers.KafkaContainer#containerIsStarting(com.github.dockerjava.api.command.InspectContainerResponse)",
+        "org.testcontainers.containers.KafkaContainer#commandKraft()",
+        "org.testcontainers.containers.KafkaContainer#commandZookeeper()",
```

Comment:
I think it is fine to remove those due to `configure`, `configureKraft` and `configureZookeeper` are still there. But, we need to be more careful about what we are exposing.

Reply:
Why weren't those `private` in the first place?

Reply:
didn't notice it in the review 😞